packages/forms/src/Forms.stories.tsx

```diff
@@ -128,6 +160,35 @@ export const SignUp = () => {
               maxLength={100}
             />
 
+            <FormikRadioGroupField fullWidth={true} label="Posts" name="post">
+              <Stack>
+                <RadioFieldCard
+                  label="Message"
+                  name="post"
```

Comment:
@umrzoq-toshkentov Why do we need name here?

Reply:
to aware of formik field and will register value to schema